depends/packages/native_qt.mk

```diff
@@ -0,0 +1,77 @@
+# Pretty much taken verbatim from Bitcoin Core
```

Comment:
I can't remember, was there a reason to stick with 5.2.1, other than it's known to work with Gitian?